tests/cases/standalone/common/tql-explain-analyze/explain.result

```diff
@@ -136,6 +179,114 @@ TQL EXPLAIN VERBOSE (0, 10, '5s') test;
 |_|_|
 +-+-+
 
+-- explain verbose at 0s, 5s and 10s. No point at 0s.
+-- SQLNESS REPLACE (-+) -
+-- SQLNESS REPLACE (\s\s+) _
+-- SQLNESS REPLACE (elapsed_compute.*) REDACTED
+-- SQLNESS REPLACE (peers.*) REDACTED
+-- SQLNESS REPLACE (RoundRobinBatch.*) REDACTED
+TQL EXPLAIN VERBOSE ('1970-01-01T00:00:00'::timestamp, '1970-01-01T00:00:00'::timestamp + '10 seconds'::interval, '5s') test;
```

Comment:
Prefer not to use `VERBOSE` in places other than testing `VERBOSE` itself. The detailed optimization procedure may change from time to time, and we don't need to track how each step is taken.